grouper/fe/templates/user.html

```diff
@@ -108,20 +108,23 @@ <h3 class="panel-title">Open Audits</h3>
                     <span aria-hidden="true">&times;</span>
                     <span class="sr-only">Close</span>
                 </button>
-                <h4 class="modal-title">Enable Group</h4>
+                <h4 class="modal-title">Enable User</h4>
            </div>
-            <div class="modal-body">
-                <p>Are you sure you want to enable this user?</p>
-            </div>
-            <div class="modal-footer">
-                <button type="button" class="btn btn-default"
-                        data-dismiss="modal">Close</button>
-                <form action="/users/{{user.id}}/enable" method="post"
-                      style="display: inline;">
+            <form action="/users/{{user.id}}/enable" method="post">
+                <div class="modal-body">
+                    <p>Are you sure you want to enable this user?</p>
+                    <label class="checkbox" style="font-weight: normal">
+                        <i>preserve any pre-existing group membership (you don't want this)</i>
+                        <input type="checkbox" name="preserve_membership" value="true"/>
```

Comment:
Doesn't this make preserving membership the default?  I assume we'd want it the other way around.